lib/LedgerNodeBlocks.js

```diff
@@ -1,18 +1,37 @@
 /*!
  * Ledger node blocks management class.
  *
- * Copyright (c) 2017-2018 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2017-2019 Digital Bazaar, Inc. All rights reserved.
  */
 'use strict';
 
-const {callbackify} = require('bedrock').util;
+const bedrock = require('bedrock');
+const {config, util: {callbackify}} = bedrock;
+const LRU = require('lru-cache');
+
+let lruCache;
```

Comment:
Maybe a more descriptive name like `blockHeightCache` would be better?  Short file, so not that big a deal.

Reply:
Done.